### DISCUSSION TITLE
vector tool bug

### DISCUSSION THREAD

**Discussion by invadergrim** [keiwando/toonsquid#421] [General]
https://github.com/keiwando/toonsquid/assets/146129527/cea7be5c-06dc-4bf3-a83c-0ec6ec9875b9

bug when using vector tools, where the selection comes up as empty

**Comment by DinoAwakening** [comment]
> 
> https://github.com/keiwando/toonsquid/assets/146129527/cea7be5c-06dc-4bf3-a83c-0ec6ec9875b9
> 
> 
> bug when using vector tools, where the selection comes up as empty 
> 

The problem there is that you have to actually close the selection before copying it. The same applies for bitmap brushes :)

**Reply by keiwando** [reply]
This is correct, but it also makes sense for the app to automatically close unfinished selections when you tap the copy button. I'll address this with the next update.
